packages/replayio/src/utils/initialization/initialize.ts

```diff
@@ -51,6 +54,8 @@ export async function initialize({
   // They also shouldn't block on failure, so we should only wait a couple of seconds before giving up
   const abortController = new AbortController();
 
+  const loggerPromise = raceWithTimeout(logger.identify(accessToken), 2_500, abortController);
```

Comment:
I think this presents a subtle race. If the timeout logic is _external_ to the logger, then it might finish after we "gave up" externally, which seems to defeat the purpose.

I think this same problem exists for our Launch Darkly initialization though so it's not like this is a _new_ thing. I'll try to give it some thought as part of PRO-797.